tutorials/plot_2_bycycle_algorithm.py

```diff
@@ -51,18 +51,15 @@
 from bycycle.cyclepoints import find_extrema, find_zerox
 from bycycle.cyclepoints.zerox import find_flank_zerox
 from bycycle.plts import plot_burst_detect_summary, plot_cyclepoints_array
+from bycycle.utils.download import load_bycycle_data
 
 pd.options.display.max_columns = 10
 
 ####################################################################################################
 
-# Simulation settings
-n_seconds = 100
+# Load data
+sig = load_bycycle_data('ca1.npy', folder='../examples/data')
```

Comment:
i think this folder can / should be just 'data'? Like it is in the `plot_1_theta` tutorial? Otherwise, doesn't this, from inside an 'examples' folder, create a en embedded folder of `examples/examples/data/`?

Reply:
If `data` is used in this case, we'll end up with the same data in two locations (`tutorials/data` and `examples/data`).  The path grabs the data from examples for tutorials.

Reply:
Oh, I didn't notice the '..' - but also, i don't think "same data in two locations" actually matters - because that's just temporary. The goal of doing this is if you download this file (independently) it can download and load the data you need. Right now, this would do so with a weird (or possible breaking) path location. 

Which actually makes me notice something: the data files should be moved to a base folder 'data' in the bycycle repo, just like in neurodsp. 

In the end, this does mean that building the whole site would lead to duplicated data files - but that's just for a temporary build. The data actually live in one place, (`bycycle/data`), and can be accessed from any tutorial / example that needs it, and I think should use a sensible / local data folder when it does so. 

Does that make sense?

Reply:
I see, I didn't think about people downloading this file independently. I'll push updates for this.

Reply:
I update `../examples/data` to `data` and moved the data folder to `bycycle/data` in the latest commit.